service/frontend/workflowHandler.go

```diff
@@ -114,6 +114,10 @@ var (
 
 const (
 	getHistoryWarnSizeLimit = 500 * 1024 // Warn when size goes over 500KB
+
+	listWorkflowFilterByID     = "WID"
```

Comment:
Can we define these under logging constants?  Basically these are well defined values for FilterType tag.